Sources/DbSearch-postgresql.php

```diff
@@ -77,10 +78,22 @@ function smf_db_search_query($identifier, $db_string, $db_values = array(), $con
 			'~RLIKE~i' => '~*',
 		),
 		'insert_log_search_results_subject' => array(
-			'~LIKE~i' => 'iLIKE',
-			'~NOT\sLIKE~i' => 'NOT iLIKE',
-			'~NOT\sRLIKE~i' => '!~*',
-			'~RLIKE~i' => '~*',
+			'/\bLIKE\b/i' => 'ILIKE',
+			'/NOT\sLIKE/i' => 'NOT ILIKE',
+			'/RLIKE/i' => '/*',
+			'/NOT RLIKE/' => '!/',
```

Comment:
/* and I/ doesn't look like valid operator: https://www.postgresql.org/docs/current/functions-matching.html

Reply:
oh right, I'll need to fix that